### DISCUSSION TITLE
Encoding special symbols with `searchParams` option

### DISCUSSION THREAD

**Discussion by Rom4ig** [sindresorhus/got#1509] [Q&A] [answered]
#### What would you like to discuss?

How i can send request without codding special symbols?

I want to send `?$format=json;metadata=full`, but when i use 
`searchParams: URLSearchParams { '$format' => 'json;metadata=full' },` 
or
 `searchParams: { '$format' : 'json;metadata=full' },`
i see this
`           '?%24format=json%3Bmetadata%3Dfull',
`

#### Checklist

- [x] I have read the documentation.

**Comment by Giotino** [comment, marked as answer]
TL; DR: With `searchParams` you can't

Explanation: `URLSearchParams` follows the WHATWG spec https://url.spec.whatwg.org/#urlsearchparams
The characters you're using are percent encoded (as you noted).
If you want to use a custom query string that doesn't follow the WHATWG spec you've to add it in the URL.

Example:
```js
got('http://example.com/?$format=json;metadata=full')
```
